app/policies/security_question_policy.rb

```diff
@@ -0,0 +1,16 @@
+class SecurityQuestionPolicy < ApplicationPolicy
+
+  def edit?
+    has_security_questions?
+  end
+
+  def update?
+    has_security_questions?
+  end
+
+  private
+
+  def has_security_questions?
+    user && !user.portal_student.nil?
+  end
```

Comment:
yeah I understand, but I still think it should be changed. It was a confusing method name even in the old code.